lib/kits/core-ui/components/paged-publication/offer-list.js

```diff
@@ -0,0 +1,200 @@
+import Mustache from 'mustache';
+import {request} from '../../../core';
+import {ComponentHelper, translate} from '../helpers';
+import './offer-list.styl';
+
+const defaultTemplate = `\
+    <div class="sgn-offers-content">
+        <div class="sgn-offers-search-container">
+            <input type="text" id="sgn-offers-search-text" class="sgn-offers-search-text" name="sgn-offers-search-text" placeholder="{{translations.searchText}}...">
+        </div>
+        <ol class="sgn-offers-ol">
+            {{#offers}}
+            <li class="sgn-offers-li">
+                <div class="sgn-offers-li-flex-container" data-page-number="{{catalog_page}}">
```

Comment:
Make sure we use publication (or paged publication or Incito publication) instead of catalog.